code/modules/clothing/head/helmet.dm

```diff
@@ -790,7 +790,7 @@
 	item_state = "ironhammer_wo_full"
 	flash_protection = FLASH_PROTECTION_MAJOR
 	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|BLOCKHAIR
-	armor_list = list(melee = 25, bullet = 60, energy = 25, bomb = 10, bio = 100, rad = 0)
+	armor_list = list(melee = 50, bullet = 50, energy = 30, bomb = 10, bio = 0, rad = 0)
```

Comment:
stealth bufffff!

Reply:
It's the same value as the dang armor - also i messed up